### ISSUE TITLE
cli skips option to create new column when created nosql storage

### ISSUE THREAD

**Issue by cityzen** [aws-amplify/amplify-cli#528]
**Describe the bug**
While going through the storage setup for dynamo, the cli will seemingly skip questions. It was defaulting to "true" for each "Would you like to add another column" and on a second try it defaulted to "false" after first column and went directly to primary key.

**To Reproduce**
1) cli: amplify storage add
2) nosql
3) Enter resource label
4) enter table name
5) name first column "name" and set type as String
6) yes to add new column
7) name second column "test" and set type as List
8) at this point mine will skip the next "would you like to add another column" and default to true and ask for the name of the new column

**Expected behavior**
Will be prompted for each column setup

**Screenshots**
![screen recording 2018-12-01 at 11 54 am](https://user-images.githubusercontent.com/345166/49330676-e35cad00-f55f-11e8-85f9-00900a91ab90.gif)

**Desktop (please complete the following information):**
 - OS: MacOS 10.14.1

**Additional context**
cli version: 0.1.34

**Comment by solshark** [comment]
Same issue with cli version 0.1.35 on macOS.

**Comment by yuth** [comment]
@cityzen what version of node do you have installed?

**Comment by solshark** [comment]
@yuth I have issue with node 10.13.0

**Comment by zjullion** [comment]
@yuth I am also experiencing the same issue with both `cli@0.2.1-multienv.4` and `cli@0.2.1-multienv.3` on node `8.11.0`.

**Comment by cityzen** [comment]
@yuth I'm on node `10.11.0`

**Comment by zjullion** [comment]
The issue is here:

https://github.com/aws-amplify/amplify-cli/blob/d5401ba3d9ce1fee19a921e3267160f7c57d6966/packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.js#L176

I suspect there is an issue with `context.prompt.confirm`, but I'm having trouble finding where that is defined.

**Comment by yuth** [comment]
@zjullion `context.prompt.confirm` is declared in [gluegun](https://github.com/infinitered/gluegun/blob/master/src/toolbox/prompt-tools.ts#L31). 

The funny things is I can reproduce this issue and one of my co-worker has identical setup and everything work on in his machine.

**Comment by zjullion** [comment]
From our testing, it doesn't always seem to return the value from the previous call - it will sometimes (seemingly randomly) return another value.

A simple work to work around this is the following:

```
    const continueQuestion = {
      type: 'confirm',
      name: 'continue',
      message: 'Wanna add another column?',
    };
    const continueAnswer = await inquirer.prompt([continueQuestion]);
    continueAttributeQuestion = continueAnswer.continue;
```

It's a little strange to me that amplify-cli hooks into gluegun's use of [enquirer] (https://github.com/enquirer/enquirer) for some prompts, while mostly using [inquirer] (https://github.com/SBoudrias/Inquirer.js/). Should all the calls to `context.prompt.confirm` maybe be refactored to inquirer calls?

**Comment by yuth** [comment]
> It's a little strange to me that amplify-cli hooks into gluegun's use of [enquirer](https://github.com/enquirer/enquirer) for some prompts, while mostly using [inquirer](https://github.com/SBoudrias/Inquirer.js/). Should all the calls to `context.prompt.confirm` maybe be refactored to inquirer calls?

Using glueguns `confirm` makes the code lot more readable. But yes we should move all our interaction code to use `inquirer` which is a little more involved as we have this in bunch of places

**Comment by zjullion** [comment]
@yuth I can open a PR for this, if you like.  It should be ready to go sometime tomorrow at the latest.

**Comment by yuth** [comment]
I already have a fix. PR coming right up in a bit

**Comment by zjullion** [comment]
You've updated the prompts in `/packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.js`, but gluegun's `context.prompt.confirm' is also used in a number of other files - is it worth updating those as well?

**Comment by yuth** [comment]
The fix has been released in `0.1.36`
